server/controller/apis/resourceController.js

```diff
@@ -289,17 +289,19 @@ exports.saveResource = async ( req, res, redirect ) => {
         }
         else if ( !req.files || Object.keys( req.files ).length === 0 ) {   // no files were uploaded       
             // no files uploaded
-            if( resource.resourceType == 1 ) {
-                this.saveResourceImage( req, res, resource.id, 'notebook-pen.svg' );
-            }
-            else if ( resource.resourceType == 2 ) {
-                this.saveResourceImage( req, res, resource.id, 'cell-molecule.svg' );
```

Comment:
As Marcus suggested. We need to revert these save image changes in resources, topics, and workspaces. They do not relate to this ticket and use the nested if structure that we talked about removing on 10/24.